metrics/protocol/shannon/metrics.go

```diff
@@ -17,14 +17,22 @@ const (
 	// The POSIX process that emits metrics
```

Comment:
Can you please add a section to the dashboard called "websocket" after we merge this in tomorrow to capture these metrics?